tests/plugins/autocomplete/autocomplete.js

```diff
@@ -89,7 +77,6 @@
 
 			editor.editable().fire( 'keyup', new CKEDITOR.dom.event( {} ) );
 
-			editable.fire( 'keydown', new CKEDITOR.dom.event( { keyCode: 40 } ) );
```

Comment:
This change is not related to this issue 🤔

Reply:
After this bug fix I just simplified above test a little. There is no need to pass arrow down keyCode because an option is selected already.

Reply:
Ah, OK, in that case it makes perfect sense - thanks for clarification!